public/index.js

```diff
@@ -316,7 +325,21 @@
 
   function addOnInputToElement(element, func){
     if (!element){ return; }
-    element.oninput = function(){ func(element.name, this.value); };
+    element.oninput = function(){
```

Comment:
Ops, I know this has not changed, but is there a reason you are not using `addEventListener('input' ..)`